sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -203,7 +203,7 @@ def test_pinjoint_arbitrary_axis():
     assert A.ang_vel_in(N).magnitude() == sqrt(omega**2)
     angle = A.ang_vel_in(N).angle_between(A.x)
     assert angle.xreplace({omega: 1}) == 0
```

Comment:
The angle between the angular velocity vector and the rotation axis should be zero. Odd that you have to subsitute omega: 1 here. Not quite sure what that is about. The angle shouldn't be a function of omega.

Reply:
`angle=acos(omega_J(t)/sqrt(omega_J(t)**2))`, so it kind of is `0`, I can replace this one with a dot-product. That is a bit more general and does the same.

Reply:
This is due to a different issue. The bottom should be `Abs(omega_J)`. See https://github.com/sympy/sympy/issues/23173

Reply:
This is fine, but it would probably reduce without subbing in 1 if it was Abs(omega_J).

Reply:
I by the way meant `cross` product. That would always just return `0` as long as they are parallel (at least in my tests).